scripts/install

```diff
@@ -144,19 +144,25 @@ exec_occ maintenance:install \
 # CONFIGURE NEXTCLOUD
 #=================================================
 
-nc_conf="${final_path}/config.json"
-cp ../conf/config.json "$nc_conf"
-ynh_replace_string "#DOMAIN#" "$domain" "$nc_conf"
-ynh_replace_string "#DATADIR#" "$datadir" "$nc_conf"
-
 # Ensure that UpdateNotification app is disabled
 exec_occ app:disable updatenotification
 
 # Enable plugins
 exec_occ app:enable user_ldap
 exec_occ ldap:create-empty-config
 
-# Load the config file in nextcloud
+# Load the installation config file in nextcloud
+nc_conf="${final_path}/config_install.json"
+cp ../conf/config_install.json "$nc_conf"
+ynh_replace_string "#DOMAIN#" "$domain" "$nc_conf"
+ynh_replace_string "#DATADIR#" "$datadir" "$nc_conf"
+exec_occ config:import "$nc_conf"
+# Then remove it
+rm -f "$nc_conf"
+
+# Load the additional config file (used also for upgrade)
```

Comment:
Why "used also for upgrade" !?
Didn't you add this file to use it only in the install script ?

Reply:
Well that's maybe not clear the way it is commented...
The first import is made on the new file config_install.
The second import is made on the file config.json which is common to install / upgrade.